src/main/java/com/lpnu/shaggybeavers/model/RelicExample.java

```diff
@@ -4,7 +4,9 @@
 import lombok.Getter;
 import lombok.Setter;
 
-@Table(name = "relic")
+import java.time.LocalDate;
+
+@Table(name = "relics")
 @Entity
 @Getter
```

Comment:
That class was as example) but okey, remove Example from name and the same problems as in classes upper